src/fdk_rdf_parser/parse_functions/concept.py

```diff
@@ -101,7 +138,9 @@ def extract_status(graph: Graph, concept_ref: URIRef) -> Optional[Dict[str, str]
     return status if len(status) > 0 else None
 
 
-def extract_target_group(graph: Graph, definition_ref: URIRef) -> Optional[str]:
+def extract_target_group_deprecated(
```

Comment:
Har du sett på https://pypi.org/project/Deprecated/?

Reply:
Jaa, vi kom vell frem til at vi ikke ville legge til en ekstra pakke kun for dette... Fant ikke noe som var innebygd

Reply:
Du kan kanskje bruke `DeprecationWarning` fra `warnings`-modulen til Python: https://docs.python.org/3/library/exceptions.html#DeprecationWarning

Reply:
```
def deprecated_func():
    warnings.warn("function is deprecated", DeprecationWarning)
    pass
```
men ville da dobbeltsjekket at den ikke kommer til å spamme loggene våre.

Reply:
I følge dokumentasjonen skal DeprecationWarnings utenfor __main__-modulen ignoreres, så tror det skal funke.

Reply:
I og med at det kun er vi som bruker dette biblioteket, virker det kanskje litt overflødig å legge til dette når navnet allerede inneholder deprecated?